doc/sources/array_api.rst

```diff
@@ -96,6 +96,10 @@ The following patched classes have support for array API inputs:
 - :obj:`sklearn.linear_model.Ridge`
 - :obj:`sklearnex.linear_model.IncrementalLinearRegression`
 - :obj:`sklearnex.linear_model.IncrementalRidge`
+- :obj:`sklearn.svm.NuSVC`
+- :obj:`sklearn.svm.NuSVR`
```

Comment:
A stupid question, but why some classes are listed here as 'sklearn.*' and the others as 'sklearnex.*'?
Shouldn't they all be 'sklearnex.*'?